client/src/components/layout/Navbar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { Button } from 'antd';
+
+import { authActions } from '../../sagas/actions';
+
+const Navbar = ({ currentUser, logout }) => {
+  return (
+    <div>
+      <Link to='/home'>Home</Link>
+      {currentUser.id && <Button onClick={logout}>Logout</Button>}
```

Comment:
🔮 What's the best way to shove the logout button to the far right?

Reply:
Probs flexbox, but as mentioned I’ve been rebuilding this with tailwind... stay tuned